src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothEventArgs.cs

```diff
@@ -560,6 +560,39 @@ public IBluetoothServerSocket Server
         }
     }
 
+    /// <summary>
+    /// An extended EventArgs class contains the socket connection requested.
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public class SocketConnectionRequestedEventArgs : EventArgs
+    {
+        internal SocketConnectionRequestedEventArgs(string remoteAddress, BluetoothSocket server)
+        {
+            RemoteAddress = remoteAddress;
+            Server = (IBluetoothServerSocket)server;
+        }
+
+        /// <summary>
+        /// The remote address.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public string RemoteAddress
+        {
+            get;
+            private set;
+        }
+
+        /// <summary>
+        /// The server socket instance.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public IBluetoothServerSocket Server
```

Comment:
Why do you pass a Server instead Client?

Let's see below use scenario
``` c#
BluetoothServerSocket.ConnectionRequested += OnConnectionRequest;

void OnConnectionRequest(object sender, SocketConnectionRequestedEventArgs  args)
{
    BluetoothServerSocket server = (BluetoothServerSocket)sender;
    ....
    // How to get handle to control client?
}

```

Reply:
No need 'Server' parameter, I will remove it.

Reply:
Accept/Reject APIs are added.